examples/colab-notebooks/colab-axolotl-example.ipynb

```diff
@@ -22,95 +22,158 @@
     "assert (torch.cuda.is_available()==True)"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "collapsed": true,
+    "id": "EMg8LT7nvW3c",
+    "outputId": "67ff16d3-33b4-470a-ab3e-406ff7c287b4"
+   },
+   "outputs": [],
+   "source": [
+    "!pip install axolotl[deepspeed]"
+   ]
+  },
   {
    "cell_type": "markdown",
    "metadata": {
-    "id": "h3nLav8oTRA5"
+    "id": "o1ELLM7zCAeW"
    },
    "source": [
-    "## Install Axolotl and dependencies"
+    "## Hugging Face login (optional)"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {
     "colab": {
-     "base_uri": "https://localhost:8080/"
+     "base_uri": "https://localhost:8080/",
+     "height": 145,
+     "referenced_widgets": [
+      "66c0fbf9f56f4e34a780237b197d0765",
+      "3f63123feba14853b8f422d7b1062123",
+      "70fa05da2afe4632bd457e08d17607db",
+      "188dbf3df94f459bb111932fc35455cb",
+      "2ea43dbac5144b69b94563fad6ec9d78",
+      "3b661f81f1d84f81b9c92abfe88424c0",
+      "feb8c4a6d07842459213f2ad4a405b5d",
+      "15ae13f194a14a529388e61b269c5576",
+      "760aa9e4dd664d2aa99ea8a70defb2a9",
+      "47705c9475104659b2d2aa4c24271db9",
+      "fd8330ea47264dbf8596f351e0b60607",
+      "53915e5377f44c5fa5054b66b8178ba4",
+      "eba8d5202f6e41899eb5133792137d9f",
+      "bdf62ad6474844f0aca9a180f36e5bd7",
+      "484dc1e12690484aa159d5dd20c6a793",
+      "59af642cd092481e91db8ff40e3cb1ce",
+      "e698618794604c21aff9501440c05eef",
+      "c9c1e2eeb3c542c4bd9793dee6cffa0f",
+      "c7615e4812804c90ab54151db742c8e6",
+      "d777ce5ab47047379f2d451bfaed0330",
+      "d5e99f1ce9974ec18bb38985fa682512",
+      "329cab0a415e42a3898d6a44fe15998c",
+      "165b5f3ad43b43d39678b44f2dcff723",
+      "ccd91efce20b475587cf450de0789735",
+      "114afd54a77a492b9df46ba1ff0f0c69",
+      "a1c0df3e05cd40a0a2b05c98475f8af9",
+      "d4b9b5462a08401aa531803c24eef1bb",
+      "7c60e10951554fb9a5dd807c30c8c6ee",
+      "2cf051bad649483caf2dcf4afee7940e",
+      "45285b02821640459f41f0ec74ea0426",
+      "bb990f12a8eb48d6a6fcff64b3df2839",
+      "32d5bdf2e00b47eea4bdd4f3eed40d16"
+     ]
     },
-    "id": "3c3yGAwnOIdi",
-    "outputId": "e3777b5a-40ef-424f-e181-62dfecd1dd01"
+    "id": "Rc4HKiV8B_TT",
+    "outputId": "9a39d2ac-7559-44b1-c6d9-dc7af8ff0a1d"
    },
-   "outputs": [],
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "66c0fbf9f56f4e34a780237b197d0765",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "VBox(children=(HTML(value='<center> <img\\nsrc=https://huggingface.co/front/assets/huggingface_logo-noborder.sv…"
+      ]
```

Comment:
is this necessary? doesn't seem to render
<img width="1064" alt="Screenshot 2024-11-21 at 10 47 44 AM" src="https://github.com/user-attachments/assets/8177c5e3-f01d-4138-b0f7-d693424771cd">

Reply:
yup, cleared those